src/components/movie_details/MovieDetails.js

```diff
@@ -2,16 +2,39 @@ import React from 'react';
 import './MovieDetails.css';
 
 const MovieDetails = ({ movie, clearSelected }) => {
+  const genres = movie.genres.map(genre => <dd key={movie.genres.indexOf(genre)} className='genre'>{genre}</dd>);
```

Comment:
this reminds me that I want to work on propTypes later today? What if movies.genres isn't an array? What if one of these values is a single genre string? :scream:
Leave this as is for now and I'll investigate a little later and introduce some propTypes

Reply:
We should absolutely add propTypes next, this line will break pretty easily... I clicked around and all seemed to be arrays with at least one string inside, but this would be a good spot for more error handling for sure!

Reply:
on it boss